modules/integrate-with-clair.adoc

```diff
@@ -3,17 +3,24 @@
 // * integration/integrate-with-image-vulnerability-scanners.adoc
 :_module-type: PROCEDURE
 [id="integrate-with-clair_{context}"]
-= Integrating with CoreOS Clair
+= Integrating with Clair
 
-You can integrate {product-title} with CoreOS Clair for the static analysis of vulnerabilities in your images.
+You can integrate {product-title} with Clair for the static analysis of vulnerabilities in your images.
+
+[NOTE]
+====
+* With {product-title-short} 3.74, you can integrate with Clair V4 Scanner.
+Red Hat has deprecated the previous CoreOS Clair integration in favor of Clair v4 integration.
+* The next RHACS 4.0 release will not support the link:https://quay.github.io/clair/concepts/authentication.html[JWT-based authentication] option for Clair V4 integration.
```

Comment:
Maybe reword this to be more like : "There is no planned support for the JWT-based authentication option for Clair V4 integration in the next RHACS 4.0 release." so it's not so definitive, since it's a future release.